rules/framework.bzl

```diff
@@ -621,6 +641,7 @@ apple_framework_packaging = rule(
         ),
         "transitive_deps": attr.label_list(
             mandatory = True,
+            cfg = apple_common.multi_arch_split,
```

Comment:
this `cfg` gives us those fragments right?

Reply:
apple_common.multi_arch_split is a 1:k transition. When we attach a transition to an attr, we need to set the transition to the cfg. This applies a 1:k outgoing transition to transitive_deps.